src/runtimes/rust/builder.js

```diff
@@ -0,0 +1,67 @@
+const { readFile } = require('fs')
+const { basename, join } = require('path')
+const { promisify } = require('util')
+
+const pReadFile = promisify(readFile)
+
+const tmp = require('tmp-promise')
+const toml = require('toml')
+
+const { lstat } = require('../../utils/fs')
+const { runCommand } = require('../../utils/shell')
+
+const { BUILD_TARGET, MANIFEST_NAME } = require('./constants')
+
+const build = async ({ srcDir }) => {
+  // We compile the binary to a temporary directory so that we don't pollute
+  // the user's functions directory.
+  const { path: targetDirectory } = await tmp.dir()
+  const functionName = basename(srcDir)
+
+  try {
+    await runCommand('cargo', ['build', '--target', BUILD_TARGET, '--release'], {
+      cwd: srcDir,
+      env: {
+        CARGO_TARGET_DIR: targetDirectory,
+      },
+    })
+  } catch (error) {
+    const hasToolchain = await checkRustToolchain()
+
+    if (!hasToolchain) {
+      throw new Error(
+        'There is no Rust toolchain installed. Visit https://ntl.fyi/missing-rust-toolchain for more information.',
```

Comment:
This would point to https://docs.netlify.com/configure-builds/manage-dependencies/#rust.